### PR TITLE
Swapping App guide

### PR DESCRIPTION
**Pull request by fernandezdavid**
Check the doc: https://docs.google.com/document/d/1U7uXPZFuMLGCUe6iRmZZ4ywf2U62ZVscox3TEJmnCHY/edit

### CODE REVIEW

File: pages/swap/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+These are the current maximum deposit amounts for this pre-release:
+
+| Asset | Amount    |
+|-------|-----------|
+| ETH   | 0.5 ETH   |
+| USDC  | 1000 USDC |
+| FLIP  | 200 FLIP  |
+| BTC   | 0,04 BTC  |
+| DOT   | 200 DOT   |
```
Comment (dandanlen) [inline pages/swap/getting-started.mdx:56]: ⬆️⬆️⬆️ 

Maybe just mention that for pre-release we have maximum deposits, and ⚠️ funds will be lost of you exceed the limits! ⚠️

File: pages/swap/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+<Callout type="warning">
+The maximum deposit amounts are in place for pre-release only. Once we launch the full version, they will be removed.
+</Callout>
+
+## Minimum deposit amounts
+
+These are the current minimum deposit amounts, required to account for protocol and gas fees:
```
Comment (dandanlen) [inline pages/swap/getting-started.mdx:64]: "Deposits below the minimum amount will be ignored and are effectively burned."

File: pages/swap/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+| FLIP  | 4 FLIP    |
+| BTC   | 0,0007 BTC|
+| DOT   | 4 DOT     |
+
+## Platform Fees
+This section covers all fees involved during a Chainflip swap in detail.
+
+### Network Fee
+
+The most important fee as its a part of the protocol burning mechanism. For every pool (USDC denominated) a swap touches, a fixed fee is taken and used to buy $FLIP and automatically burned forever. This value is currently at 0.1%.
```
Comment (dandanlen) [inline pages/swap/getting-started.mdx:79]: The network fee is only taken once per swap, not once for each pool. 

Also, Liquidity Pool fees are missing from this section. Every pool has a fixed liquidity fee that is applied. Oh, and broker fees :)

And example might help:

If you swap ETH->BTC, then you pay (in this order):

- Broker fee (zero on Chainflip's swapping app but other brokers might charge a fee)
- Ingress gas fee (Deposit/Fetch fee)
- ETH->USDC pool fee
- Network fee
- USDC->BTC fee
- Egress gas fee (Transfer fee)

**Review by dandanlen — COMMENTED** [review]
Seems a bit out of date in some places.

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**